spec/unit/lib/cloud_controller/diego/docker/lifecycle_protocol_spec.rb

```diff
@@ -118,6 +118,34 @@ module Docker
                 end
               end
             end
+
+            context 'when root user not allowed' do
```

Comment:
We may want a test to ensure that `root` user is allowed when `allow_docker_root_user` is `true`.